templates/cairo1/src/cairo_project.toml

```
@@ -1,2 +1,2 @@
 [crate_roots]
-src = "."
+$PROJECT_NAME = "."
```

```suggestion
PROJECT_NAME = "."
```

I think the dollar sign is a bit misleading here, it looks like a special character.